buoy/commands/needs.go

```diff
@@ -34,7 +34,11 @@ func addNeedsCmd(root *cobra.Command) {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			gomods := args
 
-			_, packages, err := gomod.Modules(gomods, domain)
+			selector, err := gomod.DefaultSelector(domain)
```

Comment:
This bit of logic is used 3 different places, probably should pull this out into a funtion that all 3 files can call

Reply:
EDIT: 8 different places. Should for sure be pulled out to a common function

Reply:
Hi @kvmware. I think, I do not follow. What code in specific you have in mind?

```golang
selector, err := gomod.DefaultSelector(domain)
if err != nil {
  return err
}
```

The above code couldn't really be shortened in the Golang. The syntax can't let us do it. At least that's what I'm thinking.